crypto/compactcert/builder.go

```diff
@@ -95,13 +105,13 @@ func (b *Builder) IsValid(pos uint64, sig merklesignature.Signature, verifySig b
 
```

Comment:
```
// IsValid verifies that the participant along with the signature can be inserted to the builder.
// verifySig should be set to true in production; setting it to false is useful
// for benchmarking to avoid the cost of signature checks.
```
The comment for `IsValid` above needs to be updated, and the reference to benchmarking removed.
`verifySig` is not just used for benchmarking. It is used when the signatures are loaded from the database, which are presumably already verified.